app/assets/stylesheets/friendships.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the friendships controller here.
```

Comment:
kindly delete this empty file.

Reply:
- Kindly cover the requirements from your last reviewer.

Reply:
- Kindly cover the requirements from your last reviewer.